ccm_web/client/src/components/ValidationErrorTable.tsx

```diff
@@ -80,7 +80,7 @@ function ErrorTable (props: ErrorTableProps): JSX.Element {
           <TableBody>
             {tableRows.slice(page * rowsPerPage, page * rowsPerPage + rowsPerPage).map((row) => {
               return (
-                <TableRow hover role="checkbox" tabIndex={-1} key={row.rowNumber}>
+                <TableRow hover role="checkbox" tabIndex={-1} key={row.rowNumber.toString() + row.message}>
```

Comment:
In bulk section creation it is possible to have multiple validation errors on the same line (this was not possible in the canvas gradebook validation process), so the line number was not sufficient.  This is an effective, if not pretty way to get a unique id, as I'm never creating a duplicate message.  However there is always the danger, in theory, that someone could change the code to create  duplicate messages then there could still be a warning.  I guess I would prefer to add an additional key property to the row so its always 'safe' from changes but that also increases the payload in a functionally unnecessary way so... I'm open to opinions

Reply:
Um, I see. Could you use `i` (generated index) from the `map` to make it unique, rather than the combined table row and message? Seems like that might work in both table scenarios. Thoughts?